draft-ietf-quic-recovery.md

```diff
@@ -973,24 +936,20 @@ During recovery, the congestion window remains unchanged irrespective of new
 losses or increases in the ECN-CE counter.
 
 
-## Tail Loss Probe
-
-A TLP packet MUST NOT be blocked by the sender's congestion controller. The
-sender MUST however count TLP packets against bytes in flight, since a TLP adds
-network load without establishing packet loss.
+## Probe Timeout
 
-Acknowledgement or loss of tail loss probes are treated like any other packet.
+Probe packets MUST NOT be blocked by the congestion controller.  A sender MUST
+however count these packets as being additionally in flight, since these packets
+adds network load without establishing packet loss.  Note that sending probe
+packets might cause the sender's estimated bytes in flight to exceed the
+sender's congestion window until an acknowledgement is received that establishes
```

Comment:
```suggestion
congestion window until an acknowledgement is received that establishes
```

Reply:
Also, I think this would read better as "until packets are acknowledged or lost."